datadog_checks_dev/datadog_checks/dev/_env.py

```diff
@@ -135,6 +135,15 @@ def replay_check_run(agent_collector, stub_aggregator, stub_agent):
                     data.get('device'),
                 )
 
+        network_devices_events = aggregator.get('network-devices-metadata') or []
+        for data in network_devices_events:
+            stub_aggregator.submit_network_devices_metadata_e2e(
+                check_name,
+                check_id,
+                data['UnmarshalledEvent'],
+                data['EventType'],
```

Comment:
What is the expected value(s) of this? Is it something that refers to NDM in general or is it more specific?
If the former, we can probably remove it as well.

Reply:
You mean the event type ? 

it's not that useful for now, but I think it's fine to have it in case we have more event types in the future.